SystemAlertQueue.podspec

```diff
@@ -1,7 +1,7 @@
 Pod::Spec.new do |s|
   
   s.name             = 'SystemAlertQueue'
-  s.version          = '2.1.0'
+  s.version          = '2.1.1'
```

Comment:
For this change you could increment the second digit. The third should be used for bug fixes.

Reply:
Fixed